vocabularies-gsq/borehole-status.ttl

```diff
@@ -12,22 +12,23 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 :proposed
     a skos:Concept ;
+  dcterms:references "Geoscience Australia, 2021: Borehole (Well) Status Vocabulary. <https://db-content.ga.gov.au/data_dictionary/Borehole_Status_vocabulary_2021.pdf>"@en ;
     skos:prefLabel "Proposed"@en ;
-    skos:definition "A concept or application exists to drill a borehole, but drilling has not yet commenced or been approved"@en ;
-    dcterms:provenance "Modified from: Geoscience Australia, modified from PPDM Wellbore Status vocabulary, 2014; CoalLog. Similar to PPDM's 'location' classification" ;
-    skos:usageNote "Used by GA, QLD, BOM" ;
+    skos:definition "A (pre-application) concept/intention or application exists to drill a borehole, but drilling has not yet been approved."@en ;
```

Comment:
Proposed status is a notice of intention to drill and is separate from all Landholder approvals for access and activity. I think that this definition is deviating from the actual purpose of "proposed" in Queensland Government systems and introduces ambiguity by introducing 'pre-application' into the language.

Simplifying this to something like "An intention exists to drill a borehole, but drilling has not yet commenced." would perhaps not only be more succinct, but also clearer in terms of the actual definition.